### ISSUE TITLE
Greeter does not respect displays settings

### ISSUE THREAD

**Issue by Corsseir** [elementary/greeter#65]
I can not set primary monitor before sign in. I have tried copy ~/.config/monitors.xml file to /var/lib/lightdm/.config but it did not help.

I have Dell U2414H attachted by HDMI to Dell Inspirion 5378 laptop.

**Comment by Corsseir** [comment]
I have changed pantheon greeter to unity greeter and bug still persist. Seems like /var/lib/lightdm/.config/monitors.xml are just ignored.

**Comment by DrewBe121212** [comment]
I also experienced this issue. It seems greeter simply takes whatever bios deems as the primary monitor and uses that monitor for the login screen. I was able to rectify by physically switching which monitor was plugged into port 0 of the video card, however using what is set as the primary monitor via display settings makes far more sense.

**Comment by gamunu** [comment]
I'm no expert in this subject. but Gtk 3 documentation recommends `Gdk.Display.get_primary_monitor()` over `Gdk.Screen.get_primary_monitor`. This could be the case.

https://github.com/elementary/greeter/blob/552985ec213966a89ee58e3956f8cbe8f28fbd39/src/PantheonGreeter.vala#L292

https://github.com/elementary/greeter/blob/552985ec213966a89ee58e3956f8cbe8f28fbd39/src/PantheonGreeter.vala#L281

**Comment by kt1024** [comment]
1. sudo chown lightdm:lightdm /var/lib/lightdm/.config/monistors.xml
2. append "active-monitor=0" to /etc/lightdm/pantheon-greeter.conf 
both way did NOT work.

**Comment by dannios** [comment]
I've got the same issue, as well as lightdm not respecting the screen _rotation_ as well.

**Comment by NoisyFlake** [comment]
The workaround here works: https://gist.github.com/RyanMillerC/c48d9f63f8fe386b22a7da18cf6913d2
Though this definitely should be fixed before the final release.

**Comment by lhorace** [comment]
Of course, it would be allot better that the fix mentioned by @NoisyFlake was actually incorporated in the distribution.

**Comment by andreasfelix** [comment]
How should one decide which is the primary monitor if users have set different primary monitors?

Would it maybe a solution to just show the greeter on that monitor which currently has the mouse pointer?

**Comment by BAProductions** [comment]
Try this instead I just tested it fixed for me.
Just make a symlink  ~/.config/monitors.xml in /var/lib/lightdm/.config/monitors.xml & done.
sudo rm /var/lib/lightdm/.config/monitors.xml && sudo ln -s ~/.config/monitors.xml /var/lib/lightdm/.config/monitors.xml

**Comment by BAProductions** [comment]
> How should one decide which is the primary monitor if users have set different primary monitors?
> 
> Would it maybe a solution to just show the greeter on the monitor that currently has the mouse pointer?

Try this instead I just tested it fixed for me.
Just make a symlink ~/.config/monitors.xml in /var/lib/lightdm/.config/monitors.xml & done.
sudo rm /var/lib/lightdm/.config/monitors.xml && sudo ln -s ~/.config/monitors.xml /var/lib/lightdm/.config/monitors.xml
